CAFAna/Extrap/TrivialExtrap.cxx

```diff
@@ -139,7 +147,9 @@ namespace ana
     ret->fTauFromMu     = *OscillatableSpectrum::LoadFrom(dir, "nutau_from_numu");
     ret->fTauFromMuAnti = *OscillatableSpectrum::LoadFrom(dir, "nutau_from_numu_anti");
 
-    ret->fNC = *Spectrum::LoadFrom(dir, "nc");
+    ret->fNCTot  = *Spectrum::LoadFrom(dir, "nc_tot");
+    ret->fNC     = *Spectrum::LoadFrom(dir, "nc");
+    ret->fNCAnti = *Spectrum::LoadFrom(dir, "nc_anti");
```

Comment:
My only worry is that this will make loading old prediction files impossible. We will fail to find `nc_tot` and `nc_anti`. There's no easy and safe way to gracefully handle this though. I don't think just leaving the new NC components blank could be considered safe. So this might be as good as we can get. Might just warrant a warning in #cafana. Presumably prism are willing to regenerate their files, and everyone else is using an old tag to interpret the old prediction files?

Reply:
I would go ahead and merge, and then send a warning mentioning what will happen if you update. Like I said, I suspect most people are sticking on an ancient version (unfortunately), so won't be affected.